native/dispatch.c

```diff
@@ -3342,7 +3371,7 @@ Java_com_sun_jna_Native_registerMethod(JNIEnv *env, jclass UNUSED(ncls),
   if (closure_types) (*env)->ReleaseLongArrayElements(env, closure_atypes, closure_types, 0);
   if (cvts) (*env)->ReleaseIntArrayElements(env, conversions, cvts, 0);
   data->fptr = L2A(function);
-  data->closure_rclass = (*env)->NewWeakGlobalRef(env, closure_rclass);
+  data->closure_method = (*env)->NewGlobalRef(env, closure_method);
```

Comment:
It's not enough to do it in Java_com_sun_jna_Native_unregister() as it is now? Where else does it have to be released?